.gitlab/tests/core.yml

```diff
@@ -0,0 +1,6 @@
+internal:
+  extends: .test_base_riot
+  parallel: 4
+  variables:
+    SUITE_NAME: "internal"
+    SERVICES: "testagent"
```

Comment:
is this used for anything?

it seems like we are adding the agent and test agent to every riot job?

Reply:
no, at present it's not